index.html

```diff
@@ -0,0 +1,306 @@
+    <p class="desc-category">Bedding</p>
+</a>
+
+<a href="/">
+    <img class="img-category" src="/Assets/Dogs/clothing.jpg" alt="Clothing & Accessories">
+    <p class="desc-category">Clothing & Accessories</p>
+</a>
+</div>
+
+<div class="section-part-head center">
+    <h3><span>SHOP FOR CATS</span></h3>
+</div>
+<div class="section-category">
+<a href="/">
+    <img class="img-category" src="/Assets/Cats/accessories.jpg" alt="Clothing & Accessories">
+    <p class="desc-category">Clothing & Accessories</p>
+</a>
+
+<a href="/">
+  <img class="img-category" src="/Assets/Cats/treat.jpg" alt="treat">
+  <p class="desc-category">Treat</p>
+</a>
+
+<a href="/">
+    <img class="img-category" src="/Assets/Cats/toys.jpg" alt="Toys">
+    <p class="desc-category">Toys</p>
+  </a>
+
+<a href="/">
+  <img class="img-category" src="/Assets/Cats/litter.jpg" alt="Leash & Harness">
+  <p class="desc-category">Litter</p>
+</a>
+
+<a href="/">
+  <img class="img-category" src="/Assets/Cats/grooming.jpg" alt="Grooming">
+  <p class="desc-category">Grooming</p>
+</a>
+
+<a href="/">
+    <img class="img-category" src="/Assets/Cats/bedding.jpg" alt="bedding">
+    <p class="desc-category">Bedding</p>
+</a>
+</div>
+
+
+
+<div class="section-part-head center">
+    <h3><span>SHOP FOR ESSENTIALS</span></h3>
+</div>
+<div class="section-category">
+<a href="/">
+    <img class="img-category" src="/Assets/Essentials/puppy.jpg" alt="Puppy">
+    <p class="desc-category">Puppy</p>
+</a>
+
+<a href="/">
+  <img class="img-category" src="/Assets/Essentials/Kitten.jpg" alt="Kitten">
+  <p class="desc-category">Kitten</p>
+</a>
+
+<a href="/">
+    <img class="img-category" src="/Assets/Essentials/small.jpg" alt="Small Animal">
+    <p class="desc-category">Small Pets</p>
+  </a>
+</div>
+
+<div class="section-about">
+    <img class="img-about" src="/Assets/pet.jpg" alt="Pet-about">
+    <p class="desc-about">
+        <span class="head-about"> URBAN TAILS: <br>
+        ONLINE PET STORE INDIA <br></span>
+        Urban Tails is the name that visions to comfort pets and pet parents by catering the market of pet essentials. We aim to reach at every doorstep around the globe & serve our cuddle buddies the best quality products. Our supplies range from all kinds of Food to Grooming,Clothing, Accessories, Utilities, Travel Gears & Innovative Toys. We aim to deliver what our pets crave for rather than what we want to serve. So, we make sure that our pet supplies fulfil all the standards of quality checks and protocols so that there’s no stop to joy in our little one’s life. <br>
+
+        <br> At Urban Tails - Online Pet Store India, you will find a flawless experience in searching for the right product for your pets. We are confident that your online shopping experience for your pets will be much easier than before.</p>
+</div>
+
+<div class="section-part-head center">
+    <h3><span>BRAND VALUES</span></h3>
+</div>
+<div class="section-value">
```

Comment:
You might want to add flex to either center the below images or space-around/space-between